tests/suites/0_stateless/20+_others/20_0002_planner_v2_display_error.sql

```diff
@@ -3,3 +3,5 @@ set enable_planner_v2 = 1;
 create table t(a int, b int);
 select aa from t;
 select *;
+create table t1(a int);
+select a as b from t1 intersect select b from t1 order by a;
```

Comment:
This is not exactly what I mean.

In current implementation, to support `ORDER BY` extra columns that doesn't defined in `SELECT` clause, we will pass the `ORDER BY` into `bind_select`.

But if there is set operator, then the `ORDER BY` cannot access the extra columns any more.

This test case should be rewritten as:
```sql
select a as b from t1 intersect select a as b from t1 order by a; -- column a doesn't exists, ORDER BY
```

And the implementation of order by should be fixed as well.

Reply:
fixed